FeaturedArticleWidget/FeaturedArticleWidget.swift

```diff
@@ -146,5 +151,21 @@ class FeaturedArticleWidget: UIViewController, NCWidgetProviding {
         expandedArticleView.saveButton.saveButtonState = isSaved ? .longSaved : .longSave
         collapsedArticleView.saveButton.saveButtonState = isSaved ? .longSaved : .longSave
     }
+
+    func handleTapGesture(_ tapGR: UITapGestureRecognizer) {
+        guard tapGR.state == .recognized else {
+            return
+        }
+        guard let article = self.article, let articleURL = article.url else {
+            return
+        }
+
+        let URL = articleURL as NSURL?
+        let URLToOpen = URL?.wmf_wikipediaScheme ?? NSUserActivity.wmf_baseURLForActivity(of: .explore)
+
+        self.extensionContext?.open(URLToOpen, completionHandler: { (success) in
```

Comment:
Was the leaving the completion block empty intentional? Could maybe pass it nil if nothing needs to happen on completion.